_data/Sanyam Dhawan/Sanyamdhawan99.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sanyam Dhawan # No longer than 28 characters
+institution: Punjab Engineering College 🚩 # no longer than 58 characters
+profile_pic: Sanyam_Dhawan.jpg # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
+quote: "Confusion is the root of complete clarification" # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
```

Comment:
Please follow the given instruction by removing the quotes to maintain the same format